models/src/anemoi/models/models/diffusion_encoder_processor_decoder.py

```diff
@@ -0,0 +1,809 @@
+# (C) Copyright 2024 Anemoi contributors.
```

Comment:
same comment as before - we could update to use ``Copyright 2025 Anemoi contributors.``